src/services/mongodb/collection.js

```diff
@@ -112,140 +108,97 @@ class Collection {
   aggregate(pipeline) {
     return aggregateOp(this, pipeline);
   }
+}
 
-  /**
-   * Gets the number of documents matching the filter.
-   *
-   * @param {Object} query The query used to match documents.
-   * @param {Object} options Additional find options.
-   * @param {Number} [options.limit=null] The maximum number of documents to return.
-   * @return {Number} An array of documents.
-   */
-  count(query, options = {}) {
-    return findOp(this, query, Object.assign({}, options, {
-      count: true
-    }), result => !!result.result ? result.result[0] : 0);
-  }
+// private
+function setObjectId(doc) {
```

Comment:
can remove this now.

Reply:
@mpobrien ... is that because mongodb handles missing object ids?  Just want to make sure I understand why it's not needed anymore...

Reply:
yup, the stitch server is populating them now.